backend/tests/integration/records/indexer/test_indexer_literature.py

```diff
@@ -72,3 +68,49 @@ def test_indexer_deletes_record_from_es(es_clear, db, datadir, create_record):
     hits_total = response["hits"]["total"]
 
     assert hits_total == expected_total
+
+
+@patch("inspirehep.records.api.literature.LiteratureRecord.add_files")
+def test_indexer_creates_proper_fulltext_links_in_ui_display(
+    add_file_mock, base_app, es_clear, db, create_record
```

Comment:
We have ``enable_files`` AFAIR, we can use that :)

Reply:
:+1:

Reply:
It only sets flag in config and looks like `enable_files` is a little broken, it's not restoring previous value in config  when test ends.